modules/engineering_profile_helper/modules/engineering_magazine/lib/js/magazine_navigation.js

```diff
@@ -90,13 +107,16 @@
               case 'Tab':
                 // Handle tabbing out of dropdown
                 if (e.shiftKey && index === 0) {
+                  e.preventDefault();
                   setTimeout(() => {
                     closeTopicsDropdown();
-                    toggleButton.focus();
+                    toggleButton.focus()
```

Comment:
Maybe this is a dumb question, but why did we have to remove the semicolon here?